### PR TITLE
Add link to Bias page in Debriefing section

### PR DESCRIPTION
**Pull request by bpdesigns**
## Description of changes

> Describe what this pull request is changing. Try to use plain language as much
> as possible to help teammates understand what they are reviewing.

## Known issues

> If this pull request introduces anything that needs to be addressed later,
> note it here. Some examples might be if you introduced a new link that fails
> tests but will work when another pull request is merged; you created a new
> method placeholder but it does not yet have content; or your change breaks
> something else that will be removed later. If you create any issues to
> address issues created by this pull request, this is a good place to list
> them! You can delete this section if the PR does not introduce any new issues.

## Related issues

> If this pull request is associated with any other issues or pull requests,
> lis them here. If this PR *closes* an issue, list it as `closes #issue` so
> the issue will automatically be closed when the PR is merged. If there are no
> related issues or PRs, you can delete this section.

- 
- 

## Reviewers:

> In addition to assigning reviewers in the GitHub interface, please also list
> who you would like a review from here and, if applicable, what kind of review
> you would like from them. For example, you might ask one person to review
> content while someone else reviews visual layout.

- [ ] @username: please review content
- [ ]
- [ ]

### CODE REVIEW

**Comment by bpdesigns** [comment]
update here is just adding a link.

File: _pages/research/do.md [hunk 1]
```diff
@@ -361,7 +361,7 @@ Thank you for observing! Your team appreciates your participation.
 
 ## Debriefing
 
-Debriefing is a useful way to discuss and capture what everyone took away from a session. It is important to debrief shortly after each session, while the details are still fresh in everyone’s minds. Keep in mind, debriefing is not synthesis — unless you're doing continuous synthesis, it's good to keep it descriptive only and move assessment until later in the research process.
+Debriefing is a useful way to discuss and capture what everyone took away from a session. It is important to debrief shortly after each session, while the details are still fresh in everyone’s minds. To mitigate [bias]({{ site.baseurl }}/research/bias) while debriefing using your notes from the session to ensure you are not relying solely on recall to form your impressions and insights. Keep in mind, debriefing is not synthesis — unless you're doing continuous synthesis, it's good to keep it descriptive only and move assessment until later in the research process.
```
Comment (jskinne3) [inline _pages/research/do.md:364]:
```suggestion
Debriefing is a useful way to discuss and capture what everyone took away from a session. It is important to debrief shortly after each session, while the details are still fresh in everyone’s minds. To mitigate [bias]({{ site.baseurl }}/research/bias) while debriefing, use your notes from the session to ensure you are not relying solely on recall to form your impressions and insights. Keep in mind, debriefing is not synthesis — unless you're doing continuous synthesis, it's good to keep it descriptive only and move assessment until later in the research process.
```

**Review by jskinne3 — APPROVED** [review]
The link works for me. I've made a grammar suggestion.

**Comment by bpdesigns** [comment]
Thanks @jskinne3 looks good to me.

**Pull request merged**